spec/unit/provider/oneview_san_manager_c7000_spec.rb

```diff
@@ -19,8 +19,7 @@
 provider_class = Puppet::Type.type(:oneview_san_manager).provider(:c7000)
 
 api_version = login[:api_version] || 200
-resource_name = 'SANManager'
-resourcetype = Object.const_get("OneviewSDK::API#{api_version}::C7000::#{resource_name}") unless api_version < 300
+resource_type = OneviewSDK.resource_named(:SANManager, api_version, 'C7000')
 
 describe provider_class, unit: true, if: login[:api_version] >= 300 do
```

Comment:
Why is this spec supported only `if: login[:api_version] >= 300`?
SAN Manager is supported since API 200 for the C7000 variant.